bin/deploy

```diff
@@ -11,19 +11,7 @@ set -e
 
 case "${1:-carnival-staging}" in
   carnival-staging)
-    git push --force staging master
-
-    if [ -n "$BUILD" ]; then
-      heroku run --size=PX build
-
-      git checkout -b deploy
-      git commit --allow-empty -m "Trigger deployment"
-      git push --force staging deploy:master
-      git checkout master
-      git branch -D deploy
-
-      heroku ps:scale web=1
-    fi
+    heroku docker:release
```

Comment:
We will need an `app.json` for this command to succeed. It does not replace the `Procfile`. `app.json` is used to specify add-ons. I'm miffed in general that it doesn't take the "no `app.json` present" case as "no add-ons needed", but in our case we could actually use it to specify the add-ons Carnival uses if we wanted. I've not used this feature yet.